python/oneflow/nn/graph/graph.py

```diff
@@ -592,9 +592,13 @@ def _state(self):
         for _, b in self._blocks.items():
             pa_gen = b.parameters(recurse=True)
             for pa in pa_gen:
+                if not pa.origin.is_contiguous():
```

Comment:
add a new function ` _contigious_states` like `_filter_states` and put it before `_filter_states` ?